backend/apps/github/models/user.py

```diff
@@ -1,46 +1,108 @@
-"""Github app user model."""
+"""GitHub user model and related functionality."""
 
+from django.core.exceptions import ValidationError
 from django.db import models
 
 from apps.common.models import TimestampedModel
+from apps.github.constants import (
+    GITHUB_LOGIN_REQUIRED,
+    GITHUB_NODE_ID_NOT_FOUND,
+    GITHUB_NODE_ID_REQUIRED,
+    GITHUB_USER_DATA_REQUIRED,
+)
 from apps.github.models.common import GenericUserModel, NodeModel
 
 
 class User(NodeModel, GenericUserModel, TimestampedModel):
-    """User model."""
+    """Model representing a GitHub user or organization."""
```

Comment:
We have a separate [model](https://github.com/OWASP/Nest/blob/main/backend/apps/github/models/organization.py) for GitHub organizations